forc-plugins/forc-crypto/src/address.rs

```diff
@@ -4,11 +4,16 @@ use fuels_core::types::bech32::Bech32Address;
 use serde_json::json;
 use std::str::{from_utf8, FromStr};
 
+forc::cli_examples! {
+    [ Convert any valid address to all other supported formats => crypto "address fuel12e0xwx34nfp7jrzvn9mp5qkac3yvp7h8fx37ghl7klf82vv2wkys6wd523" ]
+}
+
 #[derive(Debug, clap::Args)]
 #[clap(
     author,
     version,
-    about = "Converts any valid address to all supported formats"
+    about = "Converts any valid address to all supported formats",
```

Comment:
```suggestion
    about = "Converts an address to another format",
```

Given my comment above, if it prints all conversions then consider changing `another format` to `other formats`. Although that is vague so perhaps we ought to specify which formats it will convert into.